scripts/json2src.py

```diff
@@ -75,6 +75,9 @@ def strip_loader_meta(meta):
         if args.sections == None or config['name'] in args.sections:
             if args.lib:
                 generate_code.generate_lib(srcpath, config['name'], config['namespace'], config['tags'], args.ver, specs, input['meta'])
+            if args.common:
+                generate_code.generate_common(srcpath, config['name'], config['namespace'], config['tags'], args.ver, specs, input['meta'])
+
             # From here only generate code for functions adapters can implement.
             strip_specs_class(specs, "Loader")
```

Comment:
Ideally, this wouldn't exist. For now, I think your change is OK.

Reply:
Ok, I fixed this in #779

Reply:
Thanks 🙏 - I've approved it. I'll let you merge first and I'll rebase ontop of your changes.